doc/source/api.rst

```diff
@@ -851,6 +851,24 @@ Sparse
    SparseSeries.to_coo
    SparseSeries.from_coo
 
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_attribute.rst
+
+   Series.sparse.npoints
+   Series.sparse.kind
```

Comment:
nbytes?

Reply:
That should already be reflected in Series.nbytes.